multi_import/object_cache.py

```diff
@@ -32,37 +32,73 @@ def __init__(self, lookup_fields):
     def _default_factory(self):
         return defaultdict(set)
 
-    def _add_lookup_value(self, field, value, instance):
+    def is_multiple_fields(self, fields):
+        return (
+            hasattr(fields, '__iter__')
+            and not isinstance(fields, (str, bytes))
```

Comment:
OK so, I think that somehow we are not understanding each other.  I will respond to your comments and try to explain more thoroughly (but it could also be that i'm misunderstanding you): 

First, yes, I agree this is an 'advanced' use case, and that users should be able to input things as strings.  Let me clarify that I think users SHOULD still be able to input `lookup_fields = ('universal_id', 'item_id','title')`.  What I meant by my earlier comment is that *if* they want to have a 'multiple_fields' `lookup_field` then *that* field should be enforced to be a tuple.  So `lookup_fields = ('universal_id', ('item_id', 'title'))` would be the only way that a user could specify this new 'advanced' behaviour.  Any other format, like: `lookup_fields = ('universal_id', ['item_id', 'title'])` or any other fancy thing, wouldn't be accepted.  Does that make sense?